test/bigtest/tests/OrderLinesList/line-details-test.js

```diff
@@ -38,13 +41,37 @@ describe('Order lines list - Line details test', () => {
       ...line.attrs,
     });
 
+    this.server.get(ORDER_DETAIL_API, {
+      ...order.attrs,
```

Comment:
what is the point of this config? It looks the same as default in `config.js` .

Reply:
because of bigtest doesn't resolve promise `id` is undefined here
and it returns create order and doesn't search in collection as it's implemented in `config.js`

Reply:
`this.server.create` is a sync function, AFAIK

Reply:
yep, updated another places

Reply:
forgot about async ...